seeds/seed.js

```diff
@@ -0,0 +1,33 @@
+var Q = require('q');
+var mongoose = require('mongoose');
+
+var config = require('../env');
+var Restaurant = require('../models/Restaurant');
+var User = require('../models/User');
+
+var restaurants = [
+  new Restaurant({name: 'test', active: true}).save(),
```

Comment:
segun stack ne : http://stackoverflow.com/questions/11913074/mongoose-js-promises-or-how-to-manage-batch-save/17421145#17421145